src/beautifiers/csscomb.coffee

```diff
@@ -23,7 +24,12 @@ module.exports = class JSBeautify extends Beautifier
       try
         # Load from Project's .csscomb.json
         projectConfigPath = atom.project.getDirectories()?[0]?.resolve('.csscomb.json')
-        config = require(projectConfigPath) # Will throw error if does not exist
+        #config = require(projectConfigPath) # Will throw error if does not exist
+
+        stats = fs.statSync(projectConfigPath)
```

Comment:
Please do not use the `sync` form of this function.

If you could re-write this using the asynchronous function then I think this is good to review and merge.